fedimint-cli/src/client.rs

```diff
@@ -195,6 +195,11 @@ pub enum ClientCmd {
     Config,
     /// Gets the current fedimint AlephBFT session count
     SessionCount,
+    /// Gets the tagged fedimintd version for a peer
+    FedimintdVersion {
```

Comment:
Maybe this should be under `dev`, and maybe named `fedimint-cli dev peer-version`?